editor/src/utils/clipboard.ts

```diff
@@ -160,9 +165,10 @@ function getJSXElementPasteActions(
   const pastePostActionData: PastePostActionMenuData = {
     type: 'PASTE',
     target: target.value,
-    dataWithPropsPreserved: clipboardFirstEntry.copyDataWithPropsPreserved,
-    dataWithPropsReplaced: clipboardFirstEntry.copyDataWithPropsReplaced,
-    targetOriginalPathTrees: clipboardFirstEntry.targetOriginalContextElementPathTrees,
+    dataWithPropsPreserved: clipboardData[0].copyDataWithPropsPreserved,
```

Comment:
why not `clipboardFirstEntry`?

Reply:
good point, pushed the change in [bc1aa43](https://github.com/concrete-utopia/utopia/pull/3890/commits/bc1aa43a35548b6476d218ee1a63ff8ceaf33e66), and also refactored the surrounding code so that no `!` is needed